sdk/dotnet/S3/Inputs/BucketWebsiteArgs.cs

```diff
@@ -35,7 +35,7 @@ public sealed class BucketWebsiteArgs : Pulumi.ResourceArgs
         /// describing redirect behavior and when redirects are applied.
         /// </summary>
         [Input("routingRules")]
-        public InputUnion<string, ImmutableArray<Input<string>>>? RoutingRules { get; set; }
+        public InputUnion<string, ImmutableArray<string>>? RoutingRules { get; set; }
```

Comment:
@pgavlin, this looks suspicious, is this expected?

Reply:
yeah this is indeed breaking I believe

Reply:
Technically this is unbreaking an earlier break. I’ll find the p/p PR for context.

Reply:
I think this is the PR @pgavlin is referring to https://github.com/pulumi/pulumi/pull/7601